tools/state-copy/package.json

```diff
@@ -8,7 +8,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@polkadot/api": "^13.2.1",
-    "@polkadot/keyring": "^13.1.1"
+    "@polkadot/api": "^15.1.1",
+    "@polkadot/keyring": "^13.2.3"
```

Comment:
Did keyring not get upgrades or are we opting out of them?

Reply:
No updates currently available